### PR TITLE
Update styles on tech blog

### PR DESCRIPTION
**Pull request by darkxangel84**

### CODE REVIEW

**Comment by darkxangel84** [comment]
[Update tech blog style to match consumer site blog](https://trello.com/c/kzxxIESH/1247-update-tech-blog-style-to-match-consumer-site-blog)

**Comment by MrFirthy** [comment]
@darkxangel84 Mostly good, but a few things:

screen shot 2017-03-06 at 09 41 35

On the blog entry page, the author and job title isn't on the same black background like it is on the blog index page, making it hard to read.

screen shot 2017-03-06 at 09 41 52

It still says 'We're hiring' with a link to the careers page at the bottom of every page. Please remove this.

screen shot 2017-03-06 at 09 42 08

Can you reduce the size of the author picture to match the size it is on the consumer site.

File: _layouts/default.html [hunk 1]
```diff
@@ -8,18 +8,16 @@
...
-
     <div class="page-content">
-      <div class="wrapper">
-
+      <div class="site-header">
         <a class="site-title" href="https://octopus.energy" target="_blank"><span>{{ site.title }}</span></a>
-
         {{ content }}
       </div>
+      <div class="wrapper">
```
Comment (MrFirthy) [inline _layouts/default.html]: This shouldn't be here.

**Comment by MrFirthy** [comment]
@darkxangel84 Ok round two!

screen shot 2017-03-09 at 10 18 01

At certain screen sizes, the logo becomes full width and overlaps the back button. This should be a smaller percentage width.

screen shot 2017-03-09 at 10 18 15

The title content seems to become bigger than the header image at a certain size, meaning that it overlaps the header. This shouldn't happen.

screen shot 2017-03-09 at 10 18 34

The author section needs a bit of top margin, and for the text content to vertically align centrally in relation to the image.

**Comment by MrFirthy** [comment]
@darkxangel84 

screen shot 2017-03-10 at 15 48 15

There seems to be an issue with the author section on mobile now

screen shot 2017-03-10 at 15 48 22

It would be nice to have the arrow line up with the logo on smaller screens.

**Comment by MrFirthy** [comment]
Good job.

**Pull request merged**